### DISCUSSION TITLE
vscode: Okteto Down Logout?

### DISCUSSION THREAD

**Discussion by snowman2** [okteto/okteto#3178] [Q&A]
When I run the `okteto down` command in vscode it logs me out of my machine. This behavior is new. Is this expected?

Version information:
```
vscode: 1.70.2
Remote Kubernetes: v0.4.4
```

**Comment by pchico83** [comment]
> When I run the `okteto down` command in vscode it logs me out of my machine. This behavior is new. Is this expected?
> 
> Version information:
> 
> ```
> vscode: 1.70.2
> Remote Kubernetes: v0.4.4
> `

@snowman2 what's your OS? could you try to repro it and send us a `okteto doctor` file?
https://github.com/okteto/okteto/issues/3177#issuecomment-1286658000

**Reply by snowman2** [reply]
> what's your OS? 

Ubuntu 22

> could you try to repro it and send us a okteto doctor file?

Sure, here it is:
[okteto-doctor-20221024093934.zip](https://github.com/okteto/okteto/files/9852725/okteto-doctor-20221024093934.zip)
